Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.py

```diff
@@ -0,0 +1,614 @@
+from typing import Tuple, Dict
+from greynoise import GreyNoise, exceptions, util  # type: ignore
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+util.LOGGER.warning = util.LOGGER.debug
+
+''' CONSTANTS '''
+
+TIMEOUT = 10
+PRETTY_KEY = {
+    "ip": "IP",
+    "first_seen": "First Seen",
+    "last_seen": "Last Seen",
+    "seen": "Seen",
+    "tags": "Tags",
+    "actor": "Actor",
+    "spoofable": "Spoofable",
+    "classification": "Classification",
+    "cve": "CVE",
+    "metadata": "MetaData",
+    "asn": "ASN",
+    "city": "City",
+    "country": "Country",
+    "country_code": "Country Code",
+    "organization": "Organization",
+    "category": "Category",
+    "tor": "Tor",
+    "rdns": "RDNS",
+    "os": "OS",
+    "region": "Region",
+    "vpn": "VPN",
+    "vpn_service": "VPN Service",
+    "raw_data": "raw_data",
+    "scan": "scan",
+    "port": "port",
+    "protocol": "protocol",
+    "web": "web",
+    "paths": "paths",
+    "useragents": "useragents",
+    "ja3": "ja3",
+    "fingerprint": "fingerprint",
+    "hassh": "hassh"
+}
+IP_CONTEXT_HEADERS = ["IP", "Classification", "Actor", "CVE", "Spoofable", "VPN",
+                      "First Seen", "Last Seen"]
+API_SERVER = util.DEFAULT_CONFIG.get("api_server")
+IP_QUICK_CHECK_HEADERS = ['IP', 'Noise', 'Code', 'Code Description']
+STATS_KEY = {
+    "classifications": "Classifications",
+    "spoofable": "Spoofable",
+    "organizations": "Organizations",
+    "actors": "Actors",
+    "countries": "Countries",
+    "tags": "Tags",
+    "operating_systems": "Operating Systems",
+    "categories": "Categories",
+    "asns": "ASNs"
+}
+STATS_H_KEY = {
+    "classification": "Classification",
+    "spoofable": "Spoofable",
+    "organization": "Organization",
+    "actor": "Actor",
+    "country": "Country",
+    "tag": "Tag",
+    "operating_system": "Operating System",
+    "category": "Category",
+    "asn": "ASN",
+    "count": "Count"
+}
+QUERY_OUTPUT_PREFIX: Dict[str, str] = {
+    'IP': 'GreyNoise.IP(val.address && val.address == obj.address)',
+    'QUERY': 'GreyNoise.Query(val.query && val.query == obj.query)',
+}
+EXCEPTION_MESSAGES = {
+    'API_RATE_LIMIT': 'API Rate limit hit. Try after sometime.',
+    'UNAUTHENTICATED': 'Unauthenticated. Check the configured API Key.',
+    'COMMAND_FAIL': 'Failed to execute {} command.\n Error: {}',
+    'SERVER_ERROR': 'The server encountered an internal error for GreyNoise and was unable to complete your request.',
+    'CONNECTION_TIMEOUT': 'Connection timed out. Check your network connectivity.',
+    'PROXY': "Proxy Error - cannot connect to proxy. Either try clearing the "
+             "'Use system proxy' check-box or check the host, "
+             "authentication details and connection details for the proxy.",
+    'INVALID_RESPONSE': 'Invalid response from GreyNoise. Response: {}',
+    'QUERY_STATS_RESPONSE': 'GreyNoise request failed. Reason: {}'
+}
+
+
+''' CLIENT CLASS '''
+
+
+class Client(GreyNoise):
```

Comment:
Why not use the BaseClient from CommonServerPythhon, if you're familiar with it?

Reply:
@guykeller 
As all the api calls are handled through GreyNoise-SDK we don't need to use BaseClient provided in CommonServerPython.